### ISSUE TITLE
Bug adding text to GridLayout introduced in v0.6

### ISSUE THREAD

**Issue by Zetison** [MakieOrg/GeoMakie.jl#201]
A bug introduced in v0.6 (and not fixed in v0.6.1) can be reproduced by
`using GLMakie, GeoMakie; fig = Figure(); layout = fig[1,1] = GridLayout(); ax = GeoMakie.GeoAxis(layout[1,1]); text!(ax, 0.0, 0.0, text = "test"); display(fig)`
with error message
`ERROR: MethodError: no method matching point_iterator(::MakieCore.Text{Tuple{Vector{Makie.GlyphCollection}}})`

**Comment by asinghvi17** [comment]
This is a Makie issue unfortunately - that particular method wasn't implemented there.  I can monkeypatch it on master but would probably open a PR to fix it there...

**Comment by asinghvi17** [comment]
For now, you can simply define:
```julia
Makie.point_iterator(::Makie.MakieCore.Text{Tuple{Vector{Makie.GlyphCollection}}}) = Point2f[]
```
and see if it helps?

**Comment by apriljunge** [comment]
Just in case someone has the same problem:
This fixes issues with brackets too

**Comment by asinghvi17** [comment]
Looks fixed to me after #207 and Makie v0.21
